keg_elements/tests/test_templates.py

```diff
@@ -0,0 +1,29 @@
+from __future__ import absolute_import
+
+import keg
+from keg_elements.forms import Form
+from pyquery import PyQuery
+from wtforms import StringField
+
+
+class TestTemplates(object):
```

Comment:
:smile: I got a little stuck, so I committed (and apparently @nZac created a PR??) in the hopes that I could get some help soon.